src/nxdoc/web-ui/web-ui-tutorials/web-ui-configuration-properties.md

```diff
@@ -0,0 +1,183 @@
+
+```xml
+<component name="org.nuxeo.web.ui.properties.contrib">
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+
+     <!-- Date (time) format -->
+    <property name="org.nuxeo.web.ui.dateFormat">LL</property>
+    <property name="org.nuxeo.web.ui.dateTimeFormat">LLL</property>
+
+    <!-- First Day Of Week -->
+    <property name="org.nuxeo.web.ui.firstDayOfWeek"></property>
+    
+  </extension>
+</component>
+```
+
+### Others
+By default, there are also two more settings defined related to [Amazon S3 Direct Upload for Web UI]({{page page='amazon-s3-direct-upload'}}) and [Page Providers]({{page page='page-providers'}}).
+The first defines if the S3 Direct Upload is used and the second will define the maximum number of results displayed by the Page Provider.
+
+```xml
+<component name="org.nuxeo.web.ui.properties.contrib">
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+
+     <!-- S3 Direct upload -->
+    <property name="org.nuxeo.web.ui.s3.useDirectUpload">${nuxeo.s3storage.useDirectUpload:=false}</property>
+
+    <!-- Max Results for Tables, Grids and Lists plugged on a Nuxeo Page Provider. Falls back on elasticsearch max result window by default. -->
+    <property name="org.nuxeo.web.ui.listingMaxItems">${org.nuxeo.elasticsearch.provider.maxResultWindow:=10000}</property>
+    
+  </extension>
+</component>
+```
+
+## Customize Settings
+As any other configuration, the best way to create new settings or change/override the default ones, is to contribute a new extension as explained here [HOWTO: Contribute to an Extension]({{page page='how-to-contribute-to-an-extension'}}).
+
+{{#> callout type='warning' }}
+
+Overriding default settings must be done in a conscious way, since it will change the default/expected behavior of the platform, or at least, of some components, depending on which settings are changed.
+
+{{/callout}}
+
+## Usage Examples
+When Web UI is build these settings will be available to be used (details [here](https://github.com/nuxeo/nuxeo-web-ui/blob/master/plugin/web-ui/addon/src/main/resources/web/nuxeo.war/ui/index.jsp#L113)).
+
+The recommend way to access and use these settings can be found in several examples already implemented by Web UI.
+
+### Nuxeo App
+```javascript
+ _computeHeaders() {
+  const headers = {
+    'translate-directoryEntry': 'label',
+  };
+
+  const fetch = (Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.fetch) || {};
+
+  // add required fetchers
+  const required = { document: ['lock'], directoryEntry: ['parent'], task: ['actors'] };
+
+  Object.keys(required).forEach((k) => {
+    fetch[k] = fetch[k] || [];
+    required[k].forEach((v) => {
+      if (!fetch[k].includes(v)) {
+        fetch[k].push(v);
+      }
+    });
+  });
+
+  // generate fetch headers
+  Object.keys(fetch).forEach((f) => {
+    headers[`fetch-${f}`] = fetch[f].join(',');
+  });
+
+  return headers;
+},
```

Comment:
It is surely important to tell that a `xxx` prop is available under `Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.xxx`.
But I am not really sure we should go that deep in term of impl details.

Reply:
I may drop the `Nuxeo App` example, but keep the `formatDate`, since it is simpler and may help people to understand how to use these configs.

Reply:
:+1: (was just my feeling anw, you can bypass it ;)

Reply:
I get what @guirenard is saying, but, as someone new to Nuxeo ecosystem, I find this more complex/complete example useful.